modules/nodes-nodes-audit-log-basic-config.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-basic-config-{context}"]
+= Configuring the Basic Audit Log
```

Comment:
I'll ask again do we really have 2 audit options in 4.x?

Consider removing this in favor of advanced auditing (renamed to auditing)